application/models/neo_artifact.py

```diff
@@ -9,6 +9,8 @@ class NeoArtifact(NeoModel):
     __primarylabel__ = "NeoArtifact"
     url = Property()
     tags = RelatedTo(NeoTag, "TAGGED_WITH")
+    label_annotations = RelatedTo(NeoTag, "ANNOTATED_WITH_LABEL")
+    text_annotations = RelatedTo(NeoTag, "ANNOTATED_WITH_TEXT")
```

Comment:
So we have three different kinds of tags? User tags, google created labels and google created text tags?
Seems fine to me, I'm not sure if this is the correct or best way to model it but lets just do it and if it's stupid we at least learned something.

Other options would still be: Double relations (every tag with TAGGED_WITH + special relation like ANNOTATED_WITH_TEXT). Single TAGGED_WITH relation + properties on the relation (but the support for that in py2neo seems a little lackluster).